config.example.yml

```diff
@@ -62,6 +62,19 @@ security_levels:
   developer:
   staff:
 
+
+# Global emotes the bot can use to display various information
+global_emotes:
```

Comment:
We should make this a map. i feel like hardcoding this is a bad idea. Like to boils down to a map anyway

Reply:
Why is it a bad idea? The IDs litterally never change

Reply:
I meant it's a bad idea to hardcode the names, not the ids.

Reply:
Oh the names aren't hardcoded, they can be literally anything